.github/workflows/lint.yml

```diff
@@ -28,6 +28,22 @@ jobs:
       - name: Lint
         run: composer run lint
 
+  php-cs-fixer:
+    name: php-cs check
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
+    - name: Set up php
+      uses: shivammathur/setup-php@master
+      with:
+        php-version: 7.4
```

Comment:
Maybe use a matrix here too ?

Reply:
running once is enough, it will always have the same result

Reply:
Still, I'd rather have a matrix so we know changing the matrix in all of our tests is the wya to go instead of blindly searching for all `php 7.4` string in the files. Matrix can be use with single entries too!

https://github.com/nextcloud/app-tutorial/blob/2a89960e279b142bcf5d6bfb6ff4b7fe3672c5ec/.github/workflows/phpunit.yml#L20-L23

Reply:
Okay, that sounds good enough to me

Reply:
Just made the adjustments and push to this branch ;)